projects/fusion-ui/components/modal/v3/components/modal-content/modal-content.component.ts

```diff
@@ -0,0 +1,12 @@
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+
+@Component({
```

Comment:
this component is the same as v1 & v3 so merge them. do this for all inner component that has duplication